README.md

```diff
@@ -130,10 +130,21 @@ It is important that you make changes to posts, sections, and markers through
 the `run` and `postEditor` API. This API allows the Mobiledoc editor to conserve
 and better understand changes being made to the post.
 
+```js
+editor.run(postEditor => {
+  const mention = postEditor.builder.createAtom("mention", "John Doe", { id: 42 });
+  // insert at current cursor position:
+  // or should the user have to grab the current position from the editor first?
+  postEditor.insert(mention);
+  // or specify a different position:
+  postEditor.insert(mention, position);
```

Comment:
Agreed. 

A `replace` API could be of use for atoms too, as a common use-case would be to insert an atom over existing content eg after typing `@bob` in `lorem ipsum @bob` you'd want to replace the mention text which had just been entered with the new atom. 

I could potentially see that being more common than inserting an atom, certainly for the @mention use-case anyway. Pretty much just syntactic sugar on top of delete & insert but would remove boilerplate.